packages/manager/src/features/CloudPulse/Alerts/CreateAlert/CreateAlertDefinition.tsx

```diff
@@ -0,0 +1,247 @@
+
+  const generalError = status?.generalError;
+  const history = useHistory();
+  const onCancel = () => {
+    history.goBack();
+  };
+
+  const generateCrumbOverrides = (pathname: string) => {
+    const pathParts = pathname.split('/').filter(Boolean);
+    const lastTwoParts = pathParts.slice(-2);
+    const fullPaths: any = [];
+
+    pathParts.forEach((_, index) => {
+      fullPaths.push('/' + pathParts.slice(0, index + 1).join('/'));
+    });
+
+    const overrides = lastTwoParts.map((part, index) => ({
+      label: part,
+      linkTo: fullPaths[pathParts.length - 2 + index],
+      position: index + 1,
+    }));
+
+    return { newPathname: '/' + lastTwoParts.join('/'), overrides };
+  };
+
+  const { newPathname, overrides } = React.useMemo(
+    () => generateCrumbOverrides(location.pathname),
+    []
+  );
+
+  const CustomErrorMessage = (props: any) => (
+    <Box sx={(theme) => ({ color: theme.color.red })}>{props.children}</Box>
+  );
+
+  return (
+    <Paper>
+      <Breadcrumb
+        crumbOverrides={overrides}
+        pathname={newPathname}
+      ></Breadcrumb>
+      <FormikProvider value={formik}>
+        <form onSubmit={handleSubmit}>
+          {generalError && (
+            <Notice
+              data-qa-error
+              key={status}
+              text={status?.generalError ?? 'An unexpected error occurred'}
+              variant="error"
+            />
+          )}
+          <Typography variant="h2">1. General Information</Typography>
+          <TextField
+            // errorText="Name error"
+            label="Name"
+            name={'name'}
+            onBlur={handleBlur}
+            onChange={handleChange}
+            value={values.name ? values.name : ''}
+          />
+          {formik.touched && formik.touched.name && errors.name ? (
+            <ErrorMessage component={CustomErrorMessage} name="name" />
+          ) : null}
+          <TextField
+            label="Description"
+            name={'description'}
+            onBlur={handleBlur}
+            onChange={handleChange}
+            optional
+            value={values.description ? values.description : ''}
+          />
+          <CloudPulseServiceSelect name={'service_type'} />
+          {formik.touched &&
+          formik.touched.service_type &&
+          errors.service_type ? (
+            <ErrorMessage component={CustomErrorMessage} name="service_type" />
+          ) : null}
+          {formik.values.service_type === 'dbaas' && (
+            <EngineOption
+              engineOptions={engineOptions ? engineOptions : []}
+              name={'engineOption'}
+            />
+          )}
+          <CloudPulseRegionSelect name={'region'} />
+          {formik.touched && formik.touched.region && errors.region ? (
+            <ErrorMessage component={CustomErrorMessage} name="region" />
+          ) : null}
+          <CloudPulseMultiResourceSelect
+            cluster={values.service_type === 'dbaas' ? true : false}
```

Comment:
```suggestion
            cluster={values.service_type === 'dbaas'}
```

handle case for undefined values